### DISCUSSION TITLE
Xiaomi outdoor cams

### DISCUSSION THREAD

**Discussion by moskovskiy82** [blakeblackshear/frigate#7237] [General]
Anybody using them?
Like aw300 dirt cheap and has spec for frigate

**Comment by joekerna** [comment]
I'm also interested. Does it have RTSP?

**Comment by mtrakal** [comment]
I have two AW300, but still didn't find if it has rtsp. Looks as some proprietary stream,... but app uses ssl pinning so not easy to check on which url/port it connect to camera 😢. I still use only with xiaomi app.

**Comment by Sorcerys** [comment]
exists in protocol miot in this site https://home.miot-spec.com/spec/mxiang.camera.mod13

**Reply by mtrakal** [reply]
It's just description for IoT, but you must be verified with your Xiaomi account (like Xiaomi Home app / Google Home / Alexa with a connected account) to access the API. It don't describe where you get video stream directly from camera (and if it's evev possible)... It uses xiaomi servers to provide transcoded streams...

**Reply by Bhoft** [reply]
I also have an aw300 currently at home and would like to get the stream directly. 
What i can say that the camera streams to the cloud so that the xiaomi home app could access the stream while on mobile. But on Wifi in the same network it connects directly to the camera.
I know that because i could still access the live video in the app even when i prohibiting internet access completly to the camera (so it couldn't call "home").

The bummer  is that the cameras "motion detection" is service dependent . You get a 3 Months trial period and afterwards you can't do nothing with the camera any more.

So if anyone find out how to get the stream directly please send a message.
